### DISCUSSION TITLE
There is no grid view

### DISCUSSION THREAD

**Discussion by AntZimin** [dineug/erd-editor#355] [General]
**Describe the bug**
Don't work, and missing icon "grid view"

**To Reproduce**
In 1.0.16 version in vs code

**Comment by dineug** [comment]
No longer supported.
To continue using in the legacy UI, please install v1.0.10.
https://github.com/dineug/erd-editor/blob/main/packages/erd-editor-vscode/CHANGELOG.md#1011
